crates/router/src/db/kafka_store.rs

```diff
@@ -1109,7 +1109,7 @@ impl PaymentAttemptInterface for KafkaStore {
 
         if let Err(er) = self
             .kafka_producer
-            .log_payment_attempt(&attempt, None)
+            .log_payment_attempt(&attempt, None, TenantID("default".to_string()))
```

Comment:
```suggestion
            .log_payment_attempt(&attempt, None, self.tenant_id)
```
Can we use the tenant_id stored in `KafkaStore` struct as such